src/cmd/api/Authentication.py

```diff
@@ -0,0 +1,34 @@
+import bcrypt
+import csv
+
+
+def register(username, password):
+
+    with open("../../core/storage/users.csv", 'r') as readFile:
+        reader = csv.reader(readFile, delimiter=' ')
+        for row in reader:
+            try:
+                print(row[0].split(",")[0])
+                if row[0].split(",")[0] == username:
+                    print("Username already exists")
+                    return False
+            except:
+                pass
+
+        with open("../../core/storage/users.csv", 'a', newline='') as writeFile:
+            writer = csv.writer(writeFile)
+            salt = bcrypt.gensalt()
+            hashed_password = bcrypt.hashpw(password.encode('utf-8'), salt)
+            writer.writerow([username, hashed_password])
+
+    return True
+
+def Login(username, password):
```

Comment:
you should maybe return the JSON token from here. maybe a cleaner solution instead of returning a boolean.